.github/workflows/verifyGithubActionBuilds.yml

```diff
@@ -0,0 +1,23 @@
+name: Verify Github Action Builds
+
+on:
+    pull_request:
+        types: [opened, synchronize]
+
+jobs:
+    verify:
+        runs-on: ubuntu-latest
+        steps:
+            # Checkout the PR branch's head
+            - uses: actions/checkout@v2
+              with:
+                  ref: ${{ github.head_ref }}
```

Comment:
Is this necessary now that we fixed the compile issues with version?

Reply:
Nope, I don't think so.